emsdk

```diff
@@ -1869,6 +1877,10 @@ def main():
       elif sys.argv[i] == '--disable-assertions':
         ENABLE_LLVM_ASSERTIONS = 'OFF'
         sys.argv[i] = ''
+      elif sys.argv[i] == '--enable-wasm':
+        global ENABLE_WASM
+        ENABLE_WASM = True
+	sys.argv[i] = ''
```

Comment:
You are right. I normally do Python with four spaces so I might have just messed it up. I'll fix it.